xml/schema/CRM/Funding/FundingCaseType.xml

```diff
@@ -62,6 +62,25 @@
     </html>
   </field>
 
+  <field>
+    <name>is_summary_application</name>
```

Comment:
A short survey among native English speakers showed that this should rather be called *Combined Application*, as *summary* does not really imply the multiple-ness of applications within the funding case.